### PR TITLE
Queue a task to fire configurationchange

### PR DESCRIPTION
**Pull request by beaufortfrancois**
FIX https://github.com/w3c/mediacapture-extensions/issues/86

@youennf @jan-ivar PTAL

***
<a href="https://pr-preview.s3.amazonaws.com/beaufortfrancois/mediacapture-extensions/pull/88.html" title="Last updated on Feb 23, 2023, 4:48 PM UTC (45830b7)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/mediacapture-extensions/88/fd5ae10...beaufortfrancois:45830b7.html" title="Last updated on Feb 23, 2023, 4:48 PM UTC (45830b7)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -799,8 +799,8 @@ <h2>Exposing change of MediaStreamTrack configuration</h2>
           <!-- FIXME: Export capabilities, constraints and settings so that we can use them here. -->
           <p>Update <var>track</var>'s capabilities, constraints and settings according <var>track</var>'s underlying source.</p>
         </li>
-        <li>
-          <p>[=Fire an event=] named <var>configurationchange</var> on <var>track</var>.</p>
+        <li><p>[=Queue a task=] on [=current settings object=]'s [=environment settings object/responsible event loop=] to
+          [=fire an event=] named <var>configurationchange</var> on <var>track</var>.</p>
```
Comment (youennf) [inline index.html:803]: We probably want to queue a task that would both update the track slots and fire the event synchronously, that would mean the task would run step 2, 3 and 4 of the current algorithm.

Comment (beaufortfrancois) [reply index.html]: Fixed. Thanks!

File: index.html [hunk 2]
```diff
@@ -793,14 +793,19 @@ <h2>Exposing change of MediaStreamTrack configuration</h2>
...
-        <li><p>If <var>track</var>.{{MediaStreamTrack/readyState}} is "ended", abort these steps.</p></li>
-        <li><p>If <var>track</var>'s capabilities, constraints and settings are matching <var>source</var> configuration, abort these steps.
-        <li>
-          <!-- FIXME: Export capabilities, constraints and settings so that we can use them here. -->
-          <p>Update <var>track</var>'s capabilities, constraints and settings according <var>track</var>'s underlying source.</p>
-        </li>
-        <li>
-          <p>[=Fire an event=] named <var>configurationchange</var> on <var>track</var>.</p>
+        <li><p>[=Queue a task=] on [=current settings object=]'s [=environment settings object/responsible event loop=] 
+          to perform the following steps:</p>
```
Comment (youennf) [inline index.html:797]: We could add a note stating that we waited for the track to be muted=false and the task will be run before any other task that might have put the track to muted=true.

**Review by youennf — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (beaufortfrancois) [reply index.html:797]: Done

**Comment by beaufortfrancois** [comment]
Are we good to merge this PR?

**Review by henbos — APPROVED** [review]

**Pull request merged**